### DISCUSSION TITLE
How to connect to RemoteAPP in ws2022 with wfreerdp.exe

### DISCUSSION THREAD

**Discussion by 546870573** [FreeRDP/FreeRDP#9621] [Q&A]
I want to know how to use wfreerdp.exe to connect to RemoteApp on Windows Server 2022. I am using the nightly build version of wfreerdp.exe.
The version:
`[15:28:17:503] [30708:000013e4] [INFO][com.freerdp.client.windows] - [wf_has_console]: Detected stdin=-1 -> gui mode
This is FreeRDP version 3.0.0-dev6 (dcefd4f86)`
and I use
`wfreerdp.exe /v:192.168.0.123 /d:test /u:administrator /p:!QAZ2wsx /app:program:"||PDF"`
I have two interfaces, one for FreeRDP and one for the App, but neither of them are displaying correctly. Can you help me check if the connection parameters are correct?
![1702020954817](https://github.com/FreeRDP/FreeRDP/assets/31125208/50bc484f-de27-4f89-a20a-e242d9eb1df0)

**Comment by akallabeth** [comment]
@546870573 hope you did not just post your real passwords ;) (don´t answer, just change them if yes)
as for your issue, this looks like the remote session is still logging in?
not much I can tell except this needs more details
